### ISSUE TITLE
Only allow users access to cards they are assigned on

### ISSUE THREAD

**Issue by plyaskin** [nextcloud/deck#663]
The rules of permission (show or hide) to cards for assigned users

**Comment by juliushaertl** [comment]
> Comments for cards

Will be there in the next release: #10 

> The rules of permission (show or hide) to cards for assigned users

This is a very specific use case which will could lead to a lot of confusion when users can only see specific cards, so I don't think it is worth adding.

> Assignment alert to email via Activity

Will also be there in the next release: #211

**Comment by plyaskin** [comment]
> This is a very specific use case which will could lead to a lot of confusion when users can only see specific cards, so I don't think it is worth adding.
Maybe you can to add the specific board for this case? Or the track below on a board with permission for a user some like this?
https://en.kaiten.io/kanban

**Comment by juliushaertl** [comment]
> Maybe will you add the specific board for this case? Or the track below on a board with permission for a user some like this?

Can you try to describe what you would like to achieve with this feature? As I understand it you just want cards to be visible to specific users only.

**Comment by juliushaertl** [comment]
OK, so it would be a permission on a per board level. I think we could integrate that nicely into the board permission management by adding another checkbox like "Only allow users access to cards they are assigned on". What do you think @nextcloud/deck ?

**Comment by pixelipo** [comment]
Please see my reply in https://github.com/nextcloud/deck/issues/662

Permissions system is one of the must-have "pro" features, but it's not really necessary for private ones. In order to build it, we need to sit down and work out a really good concept for it - then a UI - then implement it :)

**Comment by plyaskin** [comment]
Hi @pixelipo,
I think you can to do this for pro, but for private will enough just this checkbox with a functionality of permission :)

**Comment by pixelipo** [comment]
@plyaskin  it's not "just a checkbox" - you need to set up a whole interface. Your requirement is just one very particular usecase in the wider range of "permissions" feature. Why would it make sense to implement this single usecase now? Why is that one more important than any other permissions feature, such as "Give "write" access to a card only for admin and assignee"?

Implementing a single usecase without preparing for a wider set of planned related usecases is not a good way to do app development.

**Comment by plyaskin** [comment]
@brianbgness, I told about all the information on one board. Do you ready to share financial information of the company for all users even if you to trust all of them? Agree not everyone should know this information as some other.

**Comment by brianbgness** [comment]
Good point.  Some information within organizations should be kept confidential.  Human Resources (aka HR) is one example.  Suppose the HR department wanted to use Deck to track employee annual reviews where employees are given raises and bonuses.  In this use case, the whole board cannot be public due to the salary information.  HR would own the board and see it all.  But for everyone else, cards and the timeline would have to be filtered by assignment.  A card titled 'Decide 2018 raise for Tom' would be assigned to a group of managers and executives.  Another card 'Meet with Tom for 2018 review' would be assigned only to Tom and his manager.

Is this what you mean?

**Comment by mkrecek234** [comment]
@juliushaertl Do you consider this meaningful? I could think about contributing, as it is "just" another option for configuring a board user's access level.

**Comment by ilovefreesw** [comment]
https://github.com/nextcloud/deck/issues/6007

Same I am looking for this feature in open source kanban tools but found no where.....
